disnake/guild.py

```diff
@@ -4150,3 +4156,116 @@ async def timeout(
 
         data = await self._state.http.edit_member(self.id, user.id, reason=reason, **payload)
         return Member(data=data, guild=self, state=self._state)
+
+    # TODO: naming
+    async def automod_rules(self) -> List[AutomodRule]:
```

Comment:
`fetch_automod_rules` just in case we want to add caching of automod rules in the future